domains/frankie.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Now that's a spicy 404 page!",
+  "repo": "https://github.com/frankie-tech/frankie-tech.github.io",
+  "owner": {
+    "username": "frankie-tech",
+    "email": "26461046+frankie-tech@users.noreply.github.com"
```

We need a way to contact you in case we need to inform you of any changes. If you do not wish to share your email address, you can specify your twitter/mastodon handle or any other social media username instead.
```suggestion
  "email": "",
  "twitter": "your-twitter-handle"
```

---

Understandable, I went ahead and added my twitter handle!

---

Thanks!